tooling/openshift_labeler/README.md

```diff
@@ -0,0 +1,56 @@
+# openshift-labeler
+Tool to label openshift nodes based on the role they play in openshift cluster and generate a tooling inventory on fly by looking at the node labels.
+
+### Requirements
+- Ansible
+- OpenShift inventory
+
+### Run
+```
+$ cd openshift-labeler
+$ ansible-playbook -vv -i <openshift-inventory> openshift_label.yml
+```
+
+### Labeling of openshift nodes
+The nodes in the openshift cluster are labeled as follows:
+
+- Master and etcd are co-located - role=master_etcd, type=pbench
+- Master                         - role=master, type=pbench
+- Nodes                          - role=node, type=pbench
```

Comment:
What about infra nodes?  They just appear as regular nodes, but it is often important to make sure they are included in the pbench collection.  Our convention is labelling them with region=infra.  It's not a guarantee, but best that we have right now.   Can we automatically (or with an option) include them?

Reply:
The script looks for nodes with region=infra under the nodes group in the inventory and labels them with node_role=infra. I have updated readme to reflect it.